SysID/index.html

```diff
@@ -0,0 +1,949 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <script src="https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.js"></script>
+    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+  </head>
+  <body>
+    <div>
+      <p>Output:</p>
+      <textarea id="output" style="width: 100%; height: 500px;" rows="10" disabled></textarea>
+    </div>
+
+    <div>
+      <form id="customForm">
+        <input type="checkbox" id="tf_checkbox" name="customCheckbox">
+        <label for="transfer function">Transfer function</label><br><br>
+
+        <input type="checkbox" id="ss_checkbox" name="customCheckbox">
+        <label for="state space">State Space</label><br><br>
+      </form>
+    </div>
+
+    <!-- FOR TRANSFER FUNCTION -->
+    <div id="tf_form" style="display:none;">
+      <!-- <label for="No. of Poles">Poles:</label>                  
+      <input type="number" id="Poles" name="Poles">
+
+      <label for="No. of Zeros">Zeros:</label>
+      <input type="number" id="Zeros" name="Zeros"> -->
+
+      <label for="Input">Input Msg:</label>
+      <input type="text" id="input_name" name="input_name"><br><br>
+
+      <label for="Input">Input Field:</label>
+      <input type="text" id="input_field" name="input_field"><br><br>
+
+      <label for="Output">Output Msg:</label>
+      <input type="text" id="output_name" name="output_name"><br><br>
+
+      <label for="Output">Output Field:</label>
+      <input type="text" id="output_field" name="output_field"><br><br>
+
+      <label for="start time">Start time (s):</label>
+      <input type="number" id="starttime" name="starttime"><br><br>
+
+      <label for="end time">End time (s):</label>
+      <input type="number" id="endtime" name="endtime"><br><br>
+
+      <label for="start frequency">Start frequency (rad/sec):</label>
+      <input type="number" id="startfreq" name="startfreq"><br><br>
+
+      <label for="end frequency">End frequency (rad/sec):</label>
+      <input type="number" id="endfreq" name="endfreq"><br><br>
+
+      <label for="LPF cutoff frequency">LPF cutoff frequency (rad/sec):</label>
+      <input type="number" id="cutofffreq" name="cutofffreq"><br><br>
+
+      <label for="customNumerator">Numerator:</label>
+      <input type="text" id="customNumerator" name="customNumerator"><br><br>
+
+      <label for="customDenominator">Denominator:</label>
+      <input type="text" id="customDenominator" name="customDenominator"><br><br>
+
+      <label for="tf_params">Symbolic params:</label>
+      <input type="text" id="tf_params" name="tf_params"><br><br>
+
+      <div id="plotDiv"></div>
+
+    </div>
+
+    <!-- FOR STATE SPACE MODEL -->
```

Comment:
Add capability for user to specify start and stop frequencies
Add capability to apply LPF to the input and all outputs